fe/fe-core/src/main/java/com/starrocks/external/hive/Utils.java

```diff
@@ -165,6 +168,10 @@ public static int[] getPrecisionAndScale(String typeStr) throws DdlException {
 
     // Array string like "Array<Array<int>>"
     public static Type convertToArrayType(String typeStr) throws DdlException {
+        if (!HIVE_UNSUPPORTED_TYPES.stream().filter(typeStr.toUpperCase()::contains).collect(Collectors.toList())
```

Comment:
If typeStr is "array<brray<int>>", it will convert to array<known>. but we need to convert it to unknown.

Reply:
> If typeStr is `array<brray<int>>`, it will convert to `array<known>`. but we need to convert it to unknown.

It has been fixed, and add UT cover this case.